app/views/examples/translated/index.njk

```diff
@@ -32,14 +32,28 @@
 
 {% extends "layout.njk" %}
 
+{%block pageTitle%}Translated examples - GOV.UK Frontend{% endblock %}
+
 {% block beforeContent %}
   {{ govukBackLink({
     "href": "/"
   }) }}
 {% endblock %}
 
 {% block content %}
-  <h1 class="govuk-heading-xl">Translated</h1>
+  <h1 class="govuk-heading-xl">Translated examples</h1>
+
+  {{ govukWarningText({
+    text: "Please do not use these translations directly in your application.",
+    iconFallbackText: "Warning"
+  }) }}
+
+  <p class="govuk-body">
+    These examples only illustrate how our components display translated text.
+    We do not guarantee that they are accurate.
+  </p>
+
+  <hr class="govuk-section-break govuk-section-break--l" />
```

Comment:
Had a quick look, don't think we tend to use the XHTML-style self-closing `<hr>`

```suggestion
  <hr class="govuk-section-break govuk-section-break--l">
```